bin/adhoc-scripts/adjustMongoDocs.py

```diff
@@ -25,12 +28,16 @@
 import json
 from pymongo import MongoClient
 
+# WMCore services
+from Utils.Timers import gmtimeSeconds
+
 
 class OptionParser(object):
     "Helper class to handle script options"
 
     def __init__(self):
         "User based option parser"
+        defaultDN = '/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=cmsunified/CN=658085/CN=Robot: CMS Unified account'
```

Comment:
I would suggest to use `/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=msunmer/CN=852819/CN=Robot: WmCore Service Account`.

Reply:
ok